src/components/OverlayManager.js

```
@@ -0,0 +1,215 @@
+    list[index] = { ...prev, sheet, row, col };
+    mapping[field] = list;
+    store.set('mapping', mapping);
+  }
+
+  /* -------------------------------------------------------------------- */
+  /* Cleanup                                                              */
+  /* -------------------------------------------------------------------- */
+
+  destroy() {
+    this.unsubscribe && this.unsubscribe();
+    DraggableController.removeEventListener(OVERLAY_MOVED, this._onOverlayMoved);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)